jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -23,33 +23,820 @@ YUI.add('store', function() {
+                  <p>Creating new charms it easy. Charms can be written
+                  in your choice of language and adapting existing
+                  scripts is straightforward. You can keep new charms
+                  private, or share them back with the community.</p>
+                  <p>
+                  <a
+                    href={doc}>
+                    Learn more about writing charms&nbsp;&rsaquo;
+                  </a></p>
+              </div>
+          </div>
+      </div>);
+    },
+
+    /**
+      The content for the development tools section of the page
+
+      @method _developmentToolsSection
+      @return {Object} The contents of the section
+    */
+    _developmentToolsSection: function() {
+      return (<div className="row row--border-bottom">
+        <div className="inner-wrapper">
+            <h2>Development tools</h2>
+            <ul className="twelve-col no-bullets equal-height">
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=gitlab">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img src={this._generateIconPath('gitlab')}
+                              alt=""
+                              className="featured-entity__image" />
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Gitlab
+                            </h3>
+                            <p>by charmers</p>
+                        </div>
+                    </a>
+                </li>
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=jenkins">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img
+                              src={this._generateIconPath('jenkins')}
+                              alt=""
+                              className="featured-entity__image" />
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Jenkins
+                            </h3>
+                            <p>by Creative</p>
+                        </div>
+                    </a>
+                </li>
+            </ul>
+        </div>
+      </div>);
+    },
+
+    /**
+      The content for the featured charms and bundles section of the page
+
+      @method _featuredSection
+      @return {Object} The contents of the section
+    */
+    _featuredSection: function() {
```

Comment:
I feel like we should have a component for each 'style' of section which would simply be passed data to render out properly.